### PR TITLE
fix(locale): copyedit

### PR DESCRIPTION
**Pull request by Artoria2e5**
* Curly quotes are totally legit in English
* Ellipses arre very legit in Chinese
* Mobile phones are 行動電話
* Supporting a software thing is called 支援
* Keys in general are called 金鑰 (密鑰 is rarely used, but i gave two a
   pass because they are... secret keys)

TODO:
* Should we translate payload

### CODE REVIEW

**Comment by Tedko** [comment]
Thanks for localization. Hope we can hand this too community soon.

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
     },
     "additional_post_box__encrypted_post_pre": {
         "message": "來 Maskbook 解密這個訊息！ $1",
-        "description": "$1 must at the end of the string."
+        "description": "$1 须写在字串末。"
     },
     "additional_post_box__encrypted_failed": {
-        "message": "請將以下内容拷貝至發佈框。"
+        "message": "請將以下内容拷貝至發于佈框。"
```
Comment (Jack-Works) [inline src/_locales/zh/messages.json]: Syntax error?

File: src/_locales/zh/messages.json [hunk 2]
```diff
@@ -25,13 +25,13 @@
...
     "add_to_key_store_verifying": {
-        "message": "Maskbook 公鑰已獲取，驗證中…"
+        "message": "Maskbook 公鑰已獲取，正在驗證…"
     },
     "add_to_key_store_failed_title": {
         "message": "Maskbook 公鑰驗證失敗 ❌"
     },
     "add_to_key_store_failed_text": {
-        "message": "$1\nMaskbook 不會添加此密鑰。"
+        "message": "$1\nMaskbook 不會添加此金鑰。"
```
Comment (Jack-Works) [inline src/_locales/zh/messages.json]: Should we use this word?

Comment (Tedko) [reply src/_locales/zh/messages.json]: Let's use 金鑰. Double check with ETH Taipei people. From his post about cryptography:
`專案內的設定檔（config）最麻煩了，裡面有部署需要的金鑰或私密資訊。`

File: src/_locales/zh/messages.json (hunk 1 above)
Comment (Artoria2e5) [reply src/_locales/zh/messages.json]: ahhhhh shite

**Comment by Misaka-0x447f** [comment]
Unexpected update. Wait until master works.

**Pull request merged**